### DISCUSSION TITLE
Energy vs Utilities

### DISCUSSION THREAD

**Discussion by Porthos-development** [home-assistant/frontend#17709] [Dashboard]
Hi, 
Fist of all I want to say that I really love the UX that we have now with HA. 
We are always looking for improvements, currently in the menu we have the Energy "tab" this tab can contain a bit more than energy alone eg: water. 
Could it be an idea to change this name to Utilities?

**Comment by bramstroker** [comment]
Utilities is also widely used in software applications to indicate small tools. So I think it might lead to more confusion than it actually solves.

**Reply by Porthos-development** [reply]
I think those small tools are placed in a different menu item "Developer Tools" in the frontend.
The following public utilities could be consolidated as well(if measured):
- Water
- Phone
- Internet
- Sewage
- Waste disposal
